Ginger/Ginger/UserControlsLib/TextEditor/DocumentEditorPage.xaml.cs

```diff
@@ -90,9 +90,21 @@ public DocumentEditorPage(string FileName, bool enableEdit = true,bool RemoveToo
             }
         }
 
-        internal void ShowAsWindow()
+
+
+        internal void ShowAsWindow(string title = null)
         {
-            GingerCore.General.LoadGenericWindow(ref genWin, App.MainWindow, eWindowShowStyle.Free, this.Title, this);
+            if (string.IsNullOrEmpty(title))
+            {
+                
+                GingerCore.General.LoadGenericWindow(ref genWin, App.MainWindow, eWindowShowStyle.Free, this.Title, this);
```

Comment:
instead repeating the LoadGenericWindow line  twice, set title on start as title= this.Title and override it if provided on method param, then send title string to LoadGenericWindow

Reply:
I tried that, but there is an error if I do that: "Default parameter value for 'title' must be a compile-time constant"

Reply:
I looked up the error, and you can only use a constant value known at compile time when specifying default parameter values

Reply:
correct, i didn't mean to please the default value in the method signature but in beginning of method